pkg/lint/lint.go

```diff
@@ -24,14 +24,14 @@ import (
 )
 
 // All runs all of the available linters on the given base directory.
-func All(basedir string, values map[string]interface{}, namespace string, strict bool) support.Linter {
+func All(basedir string, values map[string]interface{}, name, namespace string, strict bool) support.Linter {
```

Comment:
more specific naming nit:
```suggestion
func All(basedir string, values map[string]interface{}, releaseName, namespace string, strict bool) support.Linter {
```